packages/modelviewer.dev/examples/index.html

```diff
@@ -0,0 +1,94 @@
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+-->
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>&lt;model-viewer&gt; Examples and Documentation</title>
+  <meta charset="utf-8">
+  <meta name="description" content="&lt;model-viewer&gt; examples and documentation">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link type="text/css" href="../styles/examples.css" rel="stylesheet" />
+  <link type="text/css" href="../styles/docs.css" rel="stylesheet" />
+  <link rel="shortcut icon" type="image/png" href="../assets/favicon.png" />
+  <script src="../node_modules/resize-observer-polyfill/dist/ResizeObserver.js"></script>
+
+  <!-- The following libraries and polyfills are recommended to maximize browser support -->
+
+  <!-- 🚨 REQUIRED: Web Components polyfill to support Edge and Firefox < 63 -->
+  <script src="../node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+
+  <!-- 💁 OPTIONAL: Intersection Observer polyfill for better performance in Safari and IE11 -->
+  <script src="../node_modules/intersection-observer/intersection-observer.js"></script>
+
+  <!-- 💁 OPTIONAL: Resize Observer polyfill improves resize behavior in non-Chrome browsers -->
+  <script src="../node_modules/resize-observer-polyfill/dist/ResizeObserver.js"></script>
+
+  <!-- 💁 OPTIONAL: The :focus-visible polyfill removes the focus ring for some input types -->
+  <script src="../node_modules/focus-visible/dist/focus-visible.js" defer></script>
+</head>
+
+<body>
+  <div class="sidebar">
+    <div class="home lockup">
+      <a href="../" class="sidebar-mv inner-home">
+        <div class="icon-button icon-modelviewer-black inner-home"></div>
+        <h1 class="inner-home">model-viewer</h1>
+      </a>
+    </div>
+    <hr class="sidebar-hr">
+    <div class="flipper">
+      <div class="inner-flipper" id="examples-flipper" onclick="switchPages('examples', 'examples')">
+        <a class="darken bolden">EXAMPLES</a>
+      </div>
+      <div class="inner-flipper">|</div>
+      <div class="inner-flipper" id="documentation-flipper" onclick="switchPages('examples', 'docs')">
+        <a class="darken">DOCUMENTATION</a>
+      </div>
+    </div>
+    <div class="categories" id="sidebar-category-container">
+    </div>
+  </div>
+
+  <!-- Start of all documentation -->
+  <div class="outer-content">
+    <div id="loading-examples-header"></div>
+    <div id="loading-examples"></div>
+  </div>
+
+  <script type="module" src="../examples/built/documentation-main.js">
+  </script>
+  <script type="module">
+    (() => { init('examples'); })();
+  </script>
+
+  <!-- Documentation-specific dependencies: -->
+  <script type="module" src="../examples/built/dependencies.js">
```

Comment:
Do you actually need any of these dependencies or model-viewer scripts to load here? I think they're already included in the example pages themselves. For that matter, maybe they should only be here and not in the example pages? Let's just make sure we only load them once if at all possible.

Reply:
they are only loaded once, I believe that only the pure html is coming from the example pages, when they are pulled over. i.e., they have to stay here because they aren't pulled in from the other files

Reply:
Okay, sounds good. Also, it's a little easier for me to read your replies if you don't resolve the conversations. It's just annoying github UX.

Reply:
roger roger, i won't resolve anything i have a full reply to